Bottom: fc878517f43af5ee9b339295005b86a6c08e5e39
Top:    90732a346985144bed557ac18ddd1e7d31460241
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-28 21:44:43 +0100

Refresh of move-preloadelementpcb-and

---

diff --git a/src/parse_y.y b/src/parse_y.y
index 86ed589..e6ce092 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -220,7 +220,7 @@ parsepcb
 		    yyData->pcb = yyPCB;
 		    yyData->LayerN = 0;
 		    layer_group_string = NULL;
-		}
+		  }
 		  element
 		  {
 		    PCBType *pcb_save = PCB;
